recipes/megahit/meta.yaml

```diff
@@ -1,22 +1,23 @@
-{% set version = "1.1.3" %}
-{% set sha256 = "b6eefdee075aaf7a8f9090e2e8b08b770caff90aa43a255e0e220d82ce71c492" %}
+{% set version = "1.2.5-beta" %}
```

Comment:
As a rule we don't ship beta versions.

Reply:
Sorry for overlooking this. In any case, I did some progressions in this recipe and I'll use it when the final version is ready.